app.ts

```diff
@@ -47,8 +59,9 @@ if (process.env.ENABLE_AUTH === 'true') {
     session({
       secret: 'keyboard cat',
```

Comment:
Not part of this pull request, but this secret is from the passport.js docs.  Does it matter?

Reply:
This secret is used to sign the cookie value, to ensure the cookie hasn't been tempered with.
So it is necessary.

That being said, we shouldn't disclose it since this repo is public! Nice catch.

Created a ticket to create a secret in GCP and use that: https://trello.com/c/OH7OJXTE/2304-create-secret-for-session-secret-in-express-api